pepys_admin/maintenance/gui.py

```diff
@@ -60,19 +57,17 @@ def __init__(self, data_store=None):
             # It won't cause problems for users, as they will access via Pepys Admin
             # where data store will be defined
             self.data_store = DataStore(
-                "", "", "", 0, ":memory:", db_type="sqlite", show_status=False, welcome_text=""
+                "", "", "", 0, "test_gui.db", db_type="sqlite", show_status=False, welcome_text=""
```

Comment:
@robintw should we delete this file in the test `cleanup`

Reply:
Yes - fixed in 19340c5d using a new pytest fixture (that is another way of doing a teardown, but doesn't require using a test class which would stop us using another feature we need there).